src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -36,6 +36,50 @@ DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwr
     return _systemFontFallback;
 }
 
+[[nodiscard]] Microsoft::WRL::ComPtr<IDWriteFontCollection1> DxFontRenderData::PackageCollection() const

```

Comment:
I'm sad that we have to do this _once per font render data_(!) or even once per renderer, because the package is entirely read-only and we shouldn't need to hit the disk this many times per new terminal page :/

I get that we do because of the factory, but is there really no way?

Reply:
I mean I could hit the disk to lookup the paths outside the render instantiation and pass that in so the collection is made off of precached paths instead of one lookup per.

Reply:
Or per our chat... magic static the path grovel and cache in a vector to stay relatively contained.